Misc/NEWS.d/next/Library/2023-12-29-13-09-44.gh-issue-84013.rUaxkr.rst

```diff
@@ -0,0 +1,3 @@
+It is now possible to import modules from the filesystem regardless of how
+the name is normalized in the filesystem. This in particular affects
+importing modules with a name that contains accented latin characters.
```

Comment:
This probably also needs a What's New entry, lets worry about that when the PR is closer to acceptance to avoid creating merge conflicts.